product-updates/knowledge-center/viewing-error-reports.html

```diff
@@ -0,0 +1,695 @@
+          <span style="font-size: 40px; position: relative; top: 37px; left: 10px; color: #005ea2">”</span></blockquote>
+
+           <div style="min-height: 22px;"></div>
+           <p>Please refer to the <a href="https://www.acf.hhs.gov/ofa/policy-guidance/final-tanf-ssp-moe-data-reporting-system-transmission-files-layouts-and-edits">Transmission File Layout documents</a> to compare your records against their expected layouts. Feel free to reach out to the data team at tanfdata@acf.hhs.gov with additional questions.</p>
+
+  <div style="min-height: 22px;"></div>
+  <h5>Errors related to invalid values for a specific item/data element:</h5>
+  <div style="min-height: 22px;"></div>
+  <p>Invalid value errors can come up when a specific item/data element has an unexpected value (e.g. a letter or a symbol was reported for the zip code field, such as: "462$1"): </p>
+    <blockquote style="margin-left: 0px; padding-left: 10px; padding-bottom: 10px; border-color: blue; border-left: solid 3px #005ea2; font-size: 17px; font-weight: 500; line-height: 20px; background-color: #ebf9fc; border-radius: 10px;"><span style="font-size: 40px; color: #005ea2; position: relative; right: 10px; padding-left: 10px;">“</span>Item 17 (ZIP code) must be numeric.
+          <span style="font-size: 40px; position: relative; top: 37px; left: 10px; color: #005ea2">”</span></blockquote>
+          <p>To remedy these type of issues, you can compare the value reported in the file for a given item number to the allowable values in the coding instructions associated to the relevant item number. The coding instructions are linked below:</p>
+          <ul>
+            <li><a href="https://www.acf.hhs.gov/ofa/policy-guidance/tribal-tanf-data-coding-instructions">Tribal TANF Coding Instructions</a></li>
+            <li><a href="https://www.acf.hhs.gov/ofa/policy-guidance/acf-ofa-pi-23-04">TANF (ACF-199) and SSP-MOE (ACF-209) Coding Instructions</a></li>
+          </ul>
+
+
+  <div style="min-height: 22px;"></div>
+  <h5>Errors related to inconsistent values for related items/data elements in the same record:</h5>
+   <div style="min-height: 22px;"></div>
+  <p>Some errors may require review of the coding instructions for multiple items (and their respective values) to determine the proper correction. In the example below, the error is communicating that Item 49 on one of the records is either 99 or was left blank, either of which is in a conflict with the value for Item 30 in the same record. This message suggests a problem with either the value of Item 49 or the value of Item 30. Refer to the coding instructions and your own data to determine which value needs to be corrected.</p>
```

Comment:
```suggestion
  <p>Some errors may require review of the coding instructions for multiple items (and their respective values) to determine the proper correction. In the example below, the error is communicating that the value reported for Item 49 is in a conflict with the value for Item 30 in the same record. This message suggests a problem with either the value of Item 49 or the value of Item 30. Refer to the coding instructions and your own data to determine which value needs to be corrected.</p>
```

i re-worded this a bit because the error message example did not communicate what is being described.